paper-api/src/main/java/org/bukkit/block/Vault.java

```diff
@@ -144,10 +144,10 @@ public interface Vault extends TileState {
      * @apiNote Vaults will only periodically scan for nearby players, so this collection may take until the next {@link #getNextStateUpdateTime() update time} to update
      * upon a player entering its range.
      *
-     * @return An unmodifiable list of connected player uuids.
+     * @return An unmodifiable set of connected player uuids.
```

Comment:
not require change the mention in the docs of collection to set? (apinote and desc)

Reply:
Updated those now, thanks